__tests__/app.test.js

```diff
@@ -283,4 +283,30 @@ describe.only("PATCH /api/articles/:article_id", ()=>{
       expect(text).toBe('Please include a vote integer')
     })
   })
+})
+describe.only("DELETE /api/comments/:comment_id", ()=>{
+  test("204: deletes comment by comment_id, returning no content", ()=>{
+    return request(app)
+    .delete('/api/comments/1')
+    .expect(204)
+    .then(({body})=>{
+      expect(body).toEqual({})
+    })
+  })
+  test("404: returns 'Not Found' when comment_id doesn't exist", ()=>{
+    return request(app)
+    .delete('/api/comments/1337')
+    .expect(404)
+    .then(({text})=>{
+      expect(text).toBe('Not Found')
+    })
+  })
+  test("400: returns 'Please provide integer for comment_id' when provided a non-integer", ()=>{
+    return request(app)
+    .delete('/api/comments/one')
+    .expect(400)
+    .then(({text})=>{
+      expect(text).toBe('Please provide integer for comment_id')
```

Comment:
This would be fine to just have "bad request" Being very explicit in your error messages may give too much away on how to interact with your api